src/CommunityToolkit.Maui/Layouts/DockLayout.shared.cs

```diff
@@ -0,0 +1,75 @@
+﻿using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.Core.Interfaces;
+using CommunityToolkit.Maui.Core.Layouts;
+using Microsoft.Maui.Layouts;
+
+namespace CommunityToolkit.Maui.Layouts;
+
+/// <summary>DockLayout positions its child elements along the edges of the layout container.</summary>
```

Comment:
Just curious, why your IDE didn't follow the pattern for adding a `summary`? Something like this

```
/// <summary>
/// Visual Options for <see cref="CommunityToolkit.Maui.Core.Views.AlertView"/>
/// </summary>
```

Can you change that easily?